hack/jenkins/e2e.sh

```diff
@@ -192,6 +192,19 @@ case ${JOB_NAME} in
     MINION_SIZE="n1-standard-1"
     NUM_MINIONS="2"
     ;;
+
+  # Runs non-flaky tests on GCE on the release-latest branch,
+  # sequentially. As a reminder, if you need to change the skip list
```

Comment:
is the release Jenkins job going to pull e2e.sh from the release branch rather than head?